mlir/test/Conversion/UBToEmitC/convert-ub-to-emitc.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: mlir-opt -convert-ub-to-emitc %s | FileCheck %s
+
+// CHECK-LABEL: func.func @poison
+func.func @poison() {
+  // CHECK: "emitc.variable"(){{.*}}() -> i32
```

Comment:
```suggestion
  // CHECK: "emitc.variable"{{.*}} -> i32
```
Nit: Like it better when less syntax is checked (applies for the two cases below as well)